rospy_tutorials/test/test-talker-listener-with-timer.launch

```diff
@@ -1,4 +1,9 @@
 <launch>
+
   <node name="talker" pkg="rospy_tutorials" type="talker_timer.py" />
-  <test test-name="talker_listener_test" pkg="rospy_tutorials" type="talker_listener_test.py" />
+
+  <test test-name="talker_listener_test_with_timer"
+        name="talker_listener_test_with_timer"
```

Comment:
So, should the `name` be specified when the node name is used in testing to resolve name of private topic name or param name?